cli/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "OpenAPI Codegen cli",
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
+  "type": "module",
```

Comment:
Since we are using `got` package, that is es6 module only, I had to upgrade this part to `type: module`